projects/ion/src/lib/popover/component/popover.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, Input } from '@angular/core';
+import { Subject } from 'rxjs';
+import { SafeAny } from '../../utils/safe-any';
+
+import { PopoverPosition } from '../../core/types/popover';
+import { IconType, IonButtonProps } from '../../core/types';
+
+export interface PopoverProps {
+  ionPopoverTitle: string;
+  ionPopoverBody: SafeAny;
+  ionPopoverActions?: IonButtonProps[];
+  ionPopoverIcon?: IconType;
+  ionPopoverIconClose?: boolean;
+  ionPopoverPosition?: PopoverPosition;
+}
```

Comment:
Types should be defined on `core/types`.

Reply:
type declared in core/types/popover successfully😉